demos/common/cpp/utils/include/utils/ocv_common.hpp

```diff
@@ -315,3 +315,28 @@ class InputTransform {
     cv::Scalar means;
     cv::Scalar stdScales;
 };
+
+class LazyVideoWriter {
+    cv::VideoWriter writer;
+    unsigned nwritten;
+public:
+    const std::string filenames;
+    const double fps;
+    const unsigned lim;
+
+    LazyVideoWriter(const std::string& filenames, double fps, unsigned lim) :
```

Comment:
Should the wrapper contain all `cv::VideoWriter` inputs? Question about `cv::VideoWriter::fourcc`*.

Reply:
I'm not sure I got the question. Demos always use MJPG, so I hid this setting